config/application.rb

```diff
@@ -12,9 +12,5 @@
 
 module ContentTagger
   class Application < Rails::Application
-    config.active_record.raise_in_transactional_callbacks = true
-    config.eager_load_paths += %W(#{config.root}/lib)
-
-    config.action_view.field_error_proc = proc { |html_tag, _| html_tag }
```

Comment:
It's all handled elsewhere:
- that `field_error_proc` has been moved to an [initializer](https://github.com/alphagov/content-tagger/pull/345/files#diff-a5458114f5f97709241748489a9041a2)
- the `Services` module has been moved to `app/lib`, meaning we don't need `/lib` on the eager_load_path, and
- `config.active_record.raise_in_transactional_callbacks = true` is deprecated in Rails 5